### DISCUSSION TITLE
Is there any guidance on completely removing bottle-imp, after running `imp -i`?

### DISCUSSION THREAD

**Discussion by lorengordon** [arkane-systems/bottle-imp#45] [General]
I installed bottle-imp and ran `imp -i`, but am still having problems getting `firefox` to run, and I would like to remove bottle-imp. Is there any guidance on ensuring it is completely removed?

**Comment by cerebrate** [comment]
`imp -i` doesn't make any persistent changes to your system; you just run it at the start of each session to initialize a few things that can't be completely automated.

_bottle-imp_ should be completely removed just by using the normal remove command of whatever package manager your system uses that you used to install it - i.e., `sudo apt uninstall bottle-imp` on Debian or Ubuntu, or as appropriate for other systems - and `rm`ing `/etc/imp.ini` if you created one. After that, close your WSL session, do a `wsl --shutdown` to make sure it got closed all the way down, and you should be good to go when you next start one.

**Reply by lorengordon** [reply]
Ahh, got it, thanks! That wasn't clear to me from the README. But that certainly makes it easy enough. Cheers!
